powergenome/generators.py

```diff
@@ -1988,6 +1988,11 @@ def create_demand_response_gen_rows(self):
                 )
                 self.demand_response_profiles[resource] = dr_profile
 
+                dr_cf = dr_profile / dr_profile.max()
+
+                for i, row in enumerate(_df.itertuples()):
+                    _df["profile"][i] = dr_cf.iloc[:, i].values
```

Comment:
No need to `enumerate` on `itertuples` if all you need is a range. `for i in range(len(_df))` is all you need here. That said, I think you can simply do this in one line: `_df["profile"] = list(dr_cf.values.T)`.